### DISCUSSION TITLE
Update TFH header icon, and what should it be?

### DISCUSSION THREAD

**Discussion by penguinmoose** [theforumhelpers/theforumhelpers.github.io#318] [Ideas]
On the TFH website, the header icon is still from the old studio. I think it would be nice if we update the icon to something like the current (new studio) thumbnail.

Since the text in the studio thumbnail is a bit small compared to the background, I'm thinking we could make it a cropped version of the studio thumbnail (example linked at the bottom). Also, the aspect ratio of the thumbnail is rectangular, so either we only crop the sides or we can make the icon rectangular as well (though that might not look very good).

Example of square icon:

<img width="187" alt="tfh square example" src="https://user-images.githubusercontent.com/67451414/179375605-21bc9818-8939-444c-a912-4f1f41f18020.png">

What do you think?

**Comment by Accio1** [comment]
I created an issue based on this instead so that we can add labels and things like that.

Issue #319
